src/components/Table/index.tsx

```diff
@@ -0,0 +1,5 @@
+export { Table } from './Table'
+export { Head } from './Head'
+export { Body } from './Body'
+export { Row } from './Row'
+export { Cell } from './Cell'
```

Comment:
[MUST] Since jsx is not used, the extension should be `.ts`

Reply:
😳